x-pack/plugins/enterprise_search/server/index.ts

```diff
@@ -19,6 +19,12 @@ export const configSchema = schema.object({
   enabled: schema.boolean({ defaultValue: true }),
   accessCheckTimeout: schema.number({ defaultValue: 5000 }),
   accessCheckTimeoutWarning: schema.number({ defaultValue: 300 }),
+  ssl: schema.object({
+    certificateAuthorities: schema.maybe(
+      schema.oneOf([schema.arrayOf(schema.string()), schema.string()])
```

Comment:
It doesn't really make sense to allow a user to specify zero certificate authorities. How about this instead?

```suggestion
      schema.oneOf([schema.arrayOf(schema.string(), { minSize: 1 }), schema.string()])
```

Reply:
Should we make the parent `ssl` config a `schema.maybe(schema.object({}))` in that case? I don't think it should necessarily be a required setting for users just kicking the tires or for local dev, right?

Reply:
Well, `certificateAuthorities` is already an optional setting. This would just prevent users from explicitly setting `certificateAuthorities: []`